tools/pythonpkg/clean.sh

```diff
@@ -14,7 +14,6 @@ rm -rf	$SCRIPTPATH/.eggs \
 		$SCRIPTPATH/duckdb.hpp \
 		$SCRIPTPATH/parquet_extension.cpp \
 		$SCRIPTPATH/parquet_extension.hpp \
-		$SCRIPTPATH/duckdb \
```

Comment:
Does this mean that the cpp duckdb source and python duckdb source and being mixed together now?

Reply:
I'm not sure I follow, I had to remove this because we now have a `duckdb` folder in the tools/pythonpkg directory which stores the top-level python module (not sure if I'm using this terminology correctly tbh)

The cpp duckdb source is now added to the `duckdb.duckdb` submodule, this change only really required a change in the `setup.py`

Reply:
Ah, I missed that you'd moved it into duckdb.duckdb.

FWIW, it's a standard convention in Python to have native module counterparts to simply have the same name as their Python native modules, but with a `_` prefix. io & _io, socket and _socket, etc